fk/luse/src/main/java/se/inera/intyg/common/luse/v1/validator/InternalDraftValidatorImpl.java

```diff
@@ -39,6 +39,16 @@
 import static se.inera.intyg.common.fkparent.model.converter.RespConstants.SUBSTANSINTAG_SVAR_JSON_ID_21;
 import static se.inera.intyg.common.fkparent.model.converter.RespConstants.UNDERLAGFINNS_SVAR_JSON_ID_3;
 import static se.inera.intyg.common.fkparent.model.converter.RespConstants.UNDERLAG_SVAR_JSON_ID_4;
+import static se.inera.intyg.common.luse.v1.model.converter.RespConstants.DIAGNOSGRUND_FOR_NY_BEDOMNING_DELSVAR_ID;
```

Comment:
See if any of the JSON_ID constants used should be changed to using the luse.v1 constants now that you have moved some to the RespConstants.

Reply:
Fixed, also done for luae_na.